src/github.com/matrix-org/dendrite/roomserver/storage/storage.go

```diff
@@ -104,15 +106,15 @@ func (d *Database) StoreEvent(event gomatrixserverlib.Event, authEventNIDs []typ
 	}, nil
 }
 
-func (d *Database) assignRoomNID(roomID string) (types.RoomNID, error) {
+func (d *Database) assignRoomNID(txn *sql.Tx, roomID string) (types.RoomNID, error) {
```

Comment:
General convention, I guess we could go back and document the other places we do this?